zuul.d/nodeset.yaml

```diff
@@ -139,3 +139,29 @@
     nodes:
       - name: controller
         label: centos-9-stream-crc-2-30-0-6xlarge
+
+- nodeset:
+    name: centos-9-crc-2-30-0-xl
+    nodes:
+      - name: controller
+        label: centos-9-stream-crc-2-30-0-xl
+
+- nodeset:
+    name: 4x-centos-9-medium
```

Comment:
Where is this used?

Reply:
Thanks Pablo, this should have been used by `cifmw-molecule-networking_mapper` which I have now updated. Thanks for checking!